.gitignore

```diff
@@ -5,5 +5,6 @@ coverage
 .buildpath
 .project
 .settings
+.idea
```

Comment:
This should be added to a [global ignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)